README.md

```diff
@@ -1,73 +1,122 @@
+This **forkable** project provides the infraestructure to build a Progressive Web App (PWA) using Scaffold-ETH 2 base features, plus PWA oriented extra features, like Push Notifications and the capability to Install the PWA on your device.
 
-- ✅ **Contract Hot Reload**: Your frontend auto-adapts to your smart contract as you edit it.
-- 🔥 **Burner Wallet & Local Faucet**: Quickly test your application with a burner wallet and local faucet.
-- 🔐 **Integration with Wallet Providers**: Connect to different wallet providers and interact with the Ethereum network.
+To learn more about base Scaffold-ETH 2 features and development guide, check our [Docs](https://docs.scaffoldeth.io/) and [Website](https://scaffoldeth.io/).
 
-![Debug Contracts tab](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/1171422a-0ce4-4203-bcd4-d2d1941d198b)
+## 🏃 Quick Start
 
-## Requirements
+To get started with Scaffold-ETH 2 PWA, follow the steps below:
 
-Before you begin, you need to install the following tools:
+### 1 . Clone this repo & install dependencies
 
-- [Node (v18 LTS)](https://nodejs.org/en/download/)
-- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
-- [Git](https://git-scm.com/downloads)
+```bash
+gh repo clone BuidlGuidl/PWA-burner-wallet
+cd PWA-burner-wallet
+yarn install
+```
 
-## Quickstart
+### 2 . Setting up Firebase
 
-To get started with Scaffold-ETH 2, follow the steps below:
+> Note : You can also use other database as well, we are using Firebase for this example checkout `packages/nextjs/database/firebase`.
 
-1. Clone this repo & install dependencies
+Copy `packages/nextjs/.env.example` into `packages/nextjs/.env` file and fill in your Firebase credentials which starts with `FB`.
 
-```
-git clone https://github.com/scaffold-eth/scaffold-eth-2.git
-cd scaffold-eth-2
-yarn install
+Checkout [this article](https://softauthor.com/add-firebase-to-javascript-web-app/#add-firebase-sdk-to-javascript-web-app) on how to get config variables from Firebase.
+
+> Hint: If it's your first time using Firebase, please read the article from the beginning, as you'll need to create a Firebase project and register a web app first.
+
+We use the Firebase config variables in our `packages/nextjs/database/firebase/config` file:
+
+```js
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: process.env.FB_API_KEY,
+  authDomain: process.env.FB_AUTH_DOMAIN,
+  projectId: process.env.FB_PROJECT_ID,
+  storageBucket: process.env.FB_STRG_BUCKET,
+  messagingSenderId: process.env.FB_MESSAGING_SENDER_ID,
+  appId: process.env.FB_APP_ID,
+};
 ```
 
-2. Run a local network in the first terminal:
+> Hint : Make sure you have read and write [rules](https://firebase.google.com/docs/firestore/security/get-started#testing_rules) allowed in Firebase.
 
+### 3 . Setting VAPID Keys
+
+Run the following command to generate Public and Private VAPID :
```

Comment:
I'd explain a bit what VAPID keys are. Suggestion:

----
VAPID keys are a public-private key pair used with Firebase Cloud Messaging (FCM) to securely identify the server sending web push notifications.

You can generate your own VAPID keys by running the following command:

Reply:
Love this definition, to the point! 

Maybe : 
"VAPID keys are a public-private key pair used ~with Firebase Cloud Messaging (FCM)~ to securely identify the server sending web push notifications."

I think we can remove FCM from it since we are not using that service 🙌

Btw initially I was planning to use the Vercel KV store but was more comfortable with firestore as DB and looking at Vercel KV it felt hard for me to query from it (might be my very little exp with Redis)

 @carletex should we use Vercel KV since other BG app uses it or any other DB which is easy to configure?

Reply:
> I think we can remove FCM from it since we are not using that service 

My bad! ChatGPT tricked me haha. You definitions looks great.

> @carletex should we use Vercel KV since other BG app uses it or any other DB which is easy to configure?

We have a bunch of stuff using firebase (SRE, BGv3)... so that's why I didn't mind using it. We can start with Firebase and maybe we can tinker with KV later on.

Thanks!